tests/platform/fw_utility/config/mellanox_config.yml

```diff
@@ -0,0 +1,13 @@
+x86_64-mlnx_lssn2700-r0: BIOS,CPLD
```

Comment:
@noaOrMlnx The fwutil supports both modular and non modular chassis platforms. Do we want to make a corresponding infrastructure to be available in py.test?